test/gen_rpc/functional_SUITE.erl

```diff
@@ -8,13 +8,7 @@
 -author("Panagiotis Papadomitsos <pj@ezgr.net>").
 
 %%% CT Macros
--include_lib("gen_rpc/include/ct.hrl").
-
-%%% Node definitions
--define(NODE, 'gen_rpc_master@127.0.0.1').
--define(SLAVE, 'gen_rpc_slave@127.0.0.1').
--define(SLAVE_IP, '127.0.0.1').
--define(SLAVE_NAME, 'gen_rpc_slave').
+-include_lib("include/ct.hrl").
```

Comment:
* note that ct.hrl is under test since it is not part of product but for testing purposes 
* gen_rpc/include/ct.hrl confuses rebar to think ct is under the project include rather under the test
* now as include_lib("test/gen_rpc/include/ct.hrl") so not to rely on rebar to do the discovering.